index.html

```diff
@@ -1,151 +1,252 @@
+          <h2>Uber Navigation</h2>
+          <ul class="list-items">
+            <li class="count">Uber</li>
+            <li>Lead Developer</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A smart assistant to make driving more safe, efficient, and fun by
+            unlocking your most expensive computer: your car.
+          </p>
+          <ul class="card-tags">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="card-btn" type="submit">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about-myself">
+      <div class="about">
+        <h2>About Myself</h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <div class="connect">LET’S CONNECT</div>
+        <ul class="about-icons">
+          <li><img src="./src/assets/images/twitter.svg" alt="Twitter" /></li>
+          <li><img src="./src/assets/images/linkedin.svg" alt="LinkedIn" /></li>
+          <li><img src="./src/assets/images/medium.svg" alt="Medium" /></li>
+          <li><img src="./src/assets/images/github.svg" alt="Github" /></li>
+          <li>
+            <img src="./src/assets/images/angellist.svg" alt="Angellist" />
+          </li>
         </ul>
+        <button class="contact-btn" type="submit">Get My Resume</button>
+      </div>
+      <ul class="langs">
+        <li class="lang-items-title">
+          Languages
+          <select></select>
+        </li>
+        <li class="lang-items">
+          <ul class="disp-row">
+            <li class="lang-item">
+              <img
+                src="./src/assets/images/Ellipse_js.svg"
+                alt="JavaScript"
+              /><span class="item-span">JavaScript</span>
+            </li>
+            <li class="lang-item">
+              <img src="./src/assets/images/Ellipse_html.svg" alt="HTML" /><span
+                class="item-span"
+                >HTML</span
+              >
+            </li>
+            <li class="lang-item">
+              <img src="./src/assets/images/Ellipse_css.svg" alt="CSS" /><span
+                class="item-span"
+                >CSS</span
+              >
+            </li>
+          </ul>
+        </li>
+        <li class="lang-items-title underline">Frameworks<span>></span></li>
+        <li class="lang-items-title underline">Skills<span>></span></li>
+      </ul>
+    </section>
+    <section class="contact">
+      <img
+        class="bg-img"
+        src="./src/assets/images/contact-bg.svg"
+        alt="background image"
+      />
+      <img
+        class="bg-shapes"
+        src="./src/assets/images/contact-shapes-bg.svg"
+        alt="background shapes"
+      />
```

Comment:
Kindly, as recommended by the previous reviewer I recommend you to put the layout images in the background of your page (use background-image for that)